baqend/_resultGeneration.ts

```diff
@@ -77,9 +77,50 @@ export async function generateTestResult(wptTestId: string, pendingTest: model.T
   // Now the test is finished with data
   pendingTest.testDataMissing = false
 
+  const wptWaterfall = await createWaterfall(wptTestId, url, isDesktop, db);
+  const wptFilmstrip = await createFilmStrip(wptTestId, url, isDesktop, db);
+  pendingTest.wptWaterfall = wptWaterfall;
+  pendingTest.wptFilmstrip = wptFilmstrip;
+
   return pendingTest
 }
 
+/**
+ * Creates the waterfall screenshot.
+ *
+ * @param wptTestId
```

Comment:
I believe we always also have a short description of the params. Maybe you can have a look at other method signatures in that class.